### ISSUE TITLE
SkyFactory3/ItsBecauseUHasNoGoodSpawn is unable to tp player to island on login.

### ISSUE THREAD

**Issue by fucnknown** [Fungie2134/AuthMod#4]
Hi, sorry to be a pest, but it seems that the movement block stops [ItsBecauseUHasNoGoodSpawn ](https://mods.curse.com/mc-mods/minecraft/254647-itsbecauseuhasnogoodspawn) from teleporting a new player to his own island when he connects for the first time.
No errors are generated.

Experimentation has shown that the island is created, however the player is not transported, and he can't issue the command himself to be transported, if the mod is disabled, then "/tpisland <SameUser>" works fine.

Perhaps a provision could be made that if a player is not registered, and a "/tpisland <SameUser>" command is received it's allowed?

Log not showing errors:
```
[00:30:57] [Server thread/INFO]: Starting minecraft server version 1.10.2
[00:31:52] [Server thread/INFO]: Loading properties
[00:31:52] [Server thread/INFO]: Default game type: SURVIVAL
[00:31:52] [Server thread/INFO]: Generating keypair
[00:31:52] [Server thread/INFO]: Starting Minecraft server on 0.0.0.0:25565
[00:31:52] [Server thread/INFO]: Using epoll channel type
[00:32:07] [Server thread/INFO]: Preparing level "world"
[00:32:08] [Server thread/INFO]: Preparing start region for level 0
[00:32:08] [Server thread/INFO]: Done (1.505s)! For help, type "help" or "?"
[00:32:35] [Server thread/INFO]: Herobrine[/192.168.1.109:49758] logged in with entity id 1 at (0.5, 9.0, 0.5)
[00:32:35] [Server thread/INFO]: Herobrine joined the game
[00:33:04] [Server thread/INFO]: Herobrine lost connection: TextComponent{text='Disconnected', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}
[00:33:04] [Server thread/INFO]: Herobrine left the game
[00:33:15] [Server thread/INFO]: Stopping the server
[00:33:15] [Server thread/INFO]: Stopping server
[00:33:15] [Server thread/INFO]: Saving players
[00:33:15] [Server thread/INFO]: Saving worlds
[00:33:15] [Server thread/INFO]: Saving chunks for level 'world'/Overworld
[00:33:15] [Server thread/INFO]: Saving chunks for level 'world'/ExtraUtils2_Quarry_Dim
[00:33:15] [Server thread/INFO]: Saving chunks for level 'world'/fakeend
[00:33:15] [Server thread/INFO]: Saving chunks for level 'world'/fakenether
[00:33:15] [Server thread/INFO]: Saving chunks for level 'world'/fakeoverworld
[00:33:15] [Server thread/INFO]: Saving chunks for level 'world'/Overworld
```

Thank you!

**Comment by Aang23** [comment]
Presently you need to be logged in to be able to be teleported...
Forge isn't done and still have some missing features... This is why this happen because I have no way to do make this different.

I will try to implement a new option wich will update freeze coordinate if they changed of more than 5 blocks.
This will be activated / desactivated in the config file, disabled by default. This release will be out today.

Thanks !

**Comment by Aang23** [comment]
Implementing the necessary for the tp allowing function is done, I just have to finish adding config file and I the release will be out.

**Comment by Aang23** [comment]
Released in V1.4.
For your using, do not forget to delete the old config file, regenerate it by starting the server, set your config ( principally allowtp=1 and see=0 if I remember ) and ther restart again to apply. You can also change text messages.

**Comment by Aang23** [comment]
I fixed a issue in V1.5 so please use it.
Thanks !

**Comment by fucnknown** [comment]
**THANK YOU** 

It worked flawlessly, the new player was sent to his island, but he was unable to use any of the tpisland or warp commands before registering.    
The customizable messages are the cherry on top.

Excellent work!

**Comment by Aang23** [comment]
Thanks !
I released a new version (1.6 ) witch fix a lot of little bugs.
If you use this one, you must regenerate the config file ( yes this is another issue, auto config file update ).
